.travis.yml

```diff
@@ -1,13 +1,31 @@
 language: perl
+# Archive::Zip currently requires sudo
+sudo: required
+os:
+  - linux
```

Comment:
Travis now supports linux and osx. I just added this for good measure as the default is just linux